runtime/ms-rest-azure/lib/cloudError.js

```diff
@@ -108,6 +114,20 @@ class CloudError extends Error {
             type: {
               name: 'Object'
             }
+          },
+          additionalInfo: {
+            required: false,
+            serializedName: 'additionalInfo',
+            type: {
+              name: 'Sequence',
+              element: {
+                required: false,
+                serializedName: 'AdditionalInfoElementType',
+                type: {
+                  name: 'Object'
```

Comment:
I believe the structure of objects inside the additionalnfo array should be like this:
```js
type: {
  name: 'Composite'
  className: 'AdditionalErrorInfo'
  modelProperties: {
    type: {
      required: true,
      serializedName: 'type',
      type: {
          name: 'String'
      }
    },
    info: {
      required: true,
      serializedName: 'info',
       type: {
         name: 'Object'
       }
    }
  }
}
```
[.net sdk runtime](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/CommonModels/AdditionalErrorInfo.cs) and [python sdk runtime](https://github.com/Azure/msrestazure-for-python/blob/master/msrestazure/azure_exceptions.py#L233) have similar type definitions.

Reply:
Thanks. Updated the definition.